packages/integrations/vercel/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@astrojs/vercel",
   "description": "Deploy your site to Vercel",
-  "version": "6.0.0",
+  "version": "6.1.0",
```

Comment:
Editing manually would not be necessary. It is automatically done based on the changeset you wrote.

Reply:
Got it, thanks. I'm new to using `changeset` so I wasn't sure.